torch/export/graph_signature.py

```diff
@@ -121,6 +121,7 @@ class OutputKind(Enum):
     USER_OUTPUT = auto()
     LOSS_OUTPUT = auto()
     BUFFER_MUTATION = auto()
+    PARAMETER_MUTATION = auto()
```

Comment:
Does this gets properly serialized and de-serialized? Curious why this doesn't trigger the error [here](https://github.com/pytorch/pytorch/blob/main/torch/_export/serde/serialize.py#L1279).  We need to add the serialize and deserialize logic as well.

BC-wise, this could be made BC-preserving i.e. a minor version upgrade. The idea is that add a new  optional field parameter_mutation in scehma.OutputSpec.  Deserializer should be automatically assign it None when de-serializing old artifact.